ecr/templates/file_format_policy.json.tpl

```diff
@@ -0,0 +1,20 @@
+{
+  "Version": "2008-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowPushPull",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::${intg_account}:role/TDRFileFormatECSExecutionRoleIntg",
+          "arn:aws:iam::${staging_account}:role/TDRFileFormatECSExecutionRoleStaging"
```

Comment:
Can we add the production account in anticipation of deploying to prod, so we don't forget?

I've just created [TDR-495](https://national-archives.atlassian.net/browse/TDR-495) so we can add reminders for things we'll need to do when deploying to production, but it would be better to do it in advance if possible.

Reply:
We can't, I tried and it checks to see if it exists and gets very upset if it doesn't.